src/modal.component.html

```diff
@@ -1,15 +1,13 @@
-<div
-  (click)="onContainerClicked($event)"
-  class="modal fade"
-  tabindex="-1"
-  [ngClass]="{'in': visibleAnimate}"
-  [ngStyle]="{'display': visible ? 'block' : 'none', 'opacity': visible ? 1 : 0}"
-  *ngIf="visible">
+<div class="modal fade"
+     role="dialog"
+     tabindex="-1"
+     [ngClass]="{'in': visibleAnimate}"
+     style="display: block;"
+     *ngIf="visible">
   <div class="modal-dialog">
     <div class="modal-content">
       <div class="modal-header">
         <ng-container *ngTemplateOutlet="header"></ng-container>
-        <button class="close" data-dismiss="modal" type="button" aria-label="Close" (click)="close()">×</button>
```

Comment:
I wonder why you got rid of this line here.

Reply:
Just one to personalize this button without rewriting/extendig the lib each time I need a different icon/action to be taken. Give some time to solve this, I'll make personalization optional (see [this](https://blog.angular-university.io/angular-ng-template-ng-container-ngtemplateoutlet/) to get an idea of the aproach)

Reply:
solved